setup/new_source.sh

```diff
@@ -85,6 +85,11 @@ setup_pubsub_topic_subscription(){
 	set +x; echo
 }
 
+echo "Setting Cloud Run options"; set -x
```

Comment:
I don't believe this is necessary.  These are set in the cloudbuild.yaml so they are consistent across environments. https://github.com/GoogleCloudPlatform/fourkeys/blob/main/bq_workers/new_source_template/cloudbuild.yaml#L32

If we want to make the region more flexible, that should be done here: https://github.com/GoogleCloudPlatform/fourkeys/blob/main/setup/new_source.sh#L57

The cloudbuild.yaml accepts a Region substitution, but the one this script is passing in is currently hardcoded.  I think it would be a great idea to make that an input instead!

Reply:
Maybe I ran the new_source.sh script in the wrong order?. When I did, I got an error that I had to fix doing both gcloud config commands.
I'll remove them if you don't think they are necessary.

Reply:
A ha! This was a bug that was introduced by the Cloud Run API a few months ago.  The command order got switched around, so the commands with `--platform managed` are failing because they're in the wrong order.